flake8_quotes/__init__.py

```diff
@@ -30,13 +30,15 @@ def __init__(self, tree, filename='(none)', builtins=None):
 
     @classmethod
     def add_options(cls, parser):
-        parser.add_option('--quotes', default='\'', action='store',
+        parser.add_option('--quotes', default='\'', action='store', dest='inline_quotes',
```

Comment:
Ah k. With respect to `dest`, it's a similar sentiment. We already have override logic to handle the configuration file case (i.e. `def parse_options`). Having an extra kwarg here (i.e. `dest`) is 1 more thing to go wrong/handle so we should consolidate them via `def parse_options`